### PR TITLE
Do two page reloads when confirming card payments, to work around err…

### PR DESCRIPTION
**Pull request by pragnesh-patel**
This is a dirty fix for getting over 2 possible error pages that are generated  when confirming payment:
1. ESocketTimedOut
2. 409 Duplicate Key value violates unique constraint Error

### CODE REVIEW

File: src/tests/citizen/claim/steps/claim.ts [hunk 1]
```diff
@@ -206,6 +211,8 @@ module.exports = {
   makeAClaimAndSubmit (email, claimantType, defendantType, enterDefendantEmail = true) {
     this.makeAClaimAndSubmitStatementOfTruth(email, claimantType, defendantType, enterDefendantEmail)
     paymentSteps.payWithWorkingCard()
+    this.reloadPage() // reload gets over the ESOCKETTIMEDOUT Error
```
Comment (timja-kainos) [inline src/tests/citizen/claim/steps/claim.ts]: Or we should fix the code =/

Comment (timja-kainos) [reply src/tests/citizen/claim/steps/claim.ts]: @geek-soft thoughts

Comment (pragnesh-patel) [reply src/tests/citizen/claim/steps/claim.ts]: @timja-kainos @geek-soft I could put this work around in for the moment, until there is a fix. Its always a pain seeing this error breaking the tests.

Comment (rlewan) [reply src/tests/citizen/claim/steps/claim.ts]: How reliable is the workaround @pragnesh-patel ?

Comment (pragnesh-patel) [reply src/tests/citizen/claim/steps/claim.ts]: Works on my machine ;)

**Pull request merged**